### PR TITLE
Remove worklets

### PR DESCRIPTION
**Pull request by tjzel**

### CODE REVIEW

**Comment by mrousavy** [comment]
👀

File: packages/react-native-worklets/src/publicGlobals.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+   * shouldn't depend on it, we keep it here
+   * mainly for backward compatibility for our users.
+   */
+  var _WORKLET: boolean | undefined;
+
+  /**
+   * This ArrayBuffer contains the memory address of `jsi::Runtime`
+   * which is the Reanimated UI runtime.
+   */
+  var _WORKLET_RUNTIME: ArrayBuffer;
```
Comment (mrousavy) [inline packages/react-native-worklets/src/publicGlobals.ts:23]: Why not `bigint`, which is uintptr_t in C++?

Comment (tjzel) [reply packages/react-native-worklets/src/publicGlobals.ts:23]: AFAIK there was no support for bigints back in the day so this was the way 😅

**Comment by tjzel** [comment]
It was replaced by many other PRs

**Pull request closed without merging**